### PR TITLE
Send code coverage information to Coveralls and Codecov

### PR DESCRIPTION
**Pull request by 0xced**
Let’s see how it goes…

https://travis-ci.org/AFNetworking/AFNetworking/jobs/93953118

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,10 +20,15 @@ before_install:
   - gem install fastlane --no-rdoc --no-ri --no-document --quiet
   - gem install cocoapods --no-rdoc --no-ri --no-document --quiet
   - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+install:
+  - sudo easy_install cpp-coveralls
```
Comment (kylef) [inline .travis.yml]: Probably need to remove `sudo: false` from line 3 if you're going to use sudo.

Comment (0xced) [inline .travis.yml]: I think the `sudo` setting doesn’t affect OS X images. https://docs.travis-ci.com/user/workers/container-based-infrastructure/ only talks about linux and container-based stuff.

**Comment by kcharwood** [comment]
Hmmm. Still trying to figure out coveralls here.

**Comment by kcharwood** [comment]
Closing this. codecov went in with #3196. We can work on coveralls later.

:beers:

**Pull request closed without merging**